src/Facade/Services/PurchaseOrderReqFacadeService.cs

```diff
@@ -0,0 +1,125 @@
+﻿namespace Linn.Purchasing.Facade.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq.Expressions;
+
+    using Linn.Common.Facade;
+    using Linn.Common.Persistence;
+    using Linn.Common.Proxy.LinnApps;
+    using Linn.Purchasing.Domain.LinnApps;
+    using Linn.Purchasing.Domain.LinnApps.PurchaseOrders;
+    using Linn.Purchasing.Domain.LinnApps.Suppliers;
+    using Linn.Purchasing.Resources;
+    using Linn.Purchasing.Resources.SearchResources;
+
+    public class PurchaseOrderReqFacadeService : FacadeFilterResourceService<PurchaseOrderReq, int,
+        PurchaseOrderReqResource, PurchaseOrderReqResource, PurchaseOrderReqSearchResource>
+    {
+        private readonly IPurchaseOrderReqService domainService;
+
+        private IDatabaseService databaseService;
+
+        public PurchaseOrderReqFacadeService(
+            IRepository<PurchaseOrderReq, int> repository,
+            ITransactionManager transactionManager,
+            IBuilder<PurchaseOrderReq> resourceBuilder,
+            IPurchaseOrderReqService domainService,
+            IDatabaseService databaseService)
+            : base(repository, transactionManager, resourceBuilder)
+        {
+            this.domainService = domainService;
+            this.databaseService = databaseService;
+        }
+
+        protected override PurchaseOrderReq CreateFromResource(
+            PurchaseOrderReqResource resource,
+            IEnumerable<string> privileges = null)
+        {
+            var newReq = this.BuildEntityFromResourceHelper(resource);
+
+            newReq.ReqNumber = this.databaseService.GetNextVal("BLUE_REQ_SEQ");
+
+            return this.domainService.Create(newReq, privileges);
+        }
+
+        protected override void DeleteOrObsoleteResource(PurchaseOrderReq entity, IEnumerable<string> privileges = null)
+        {
+            throw new NotImplementedException();
+        }
+
+        protected override Expression<Func<PurchaseOrderReq, bool>> FilterExpression(
+            PurchaseOrderReqSearchResource searchResource)
+        {
+            throw new NotImplementedException();
+        }
+
+        protected override void SaveToLogTable(
+            string actionType,
+            int userNumber,
+            PurchaseOrderReq entity,
+            PurchaseOrderReqResource resource,
+            PurchaseOrderReqResource updateResource)
+        {
+            throw new NotImplementedException();
+        }
+
+        protected override Expression<Func<PurchaseOrderReq, bool>> SearchExpression(string searchTerm)
+        {
+            return x => x.OrderNumber.ToString().Contains(searchTerm);
```

Comment:
expecting to change this search to a search on multiple things (ie filter not search) but need to check with users which fields first and haven't implemented the search page on front end yet, is a FacadeFilterResourceService in anticipation of that basically